dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/LdapFailoverTest.java

```diff
@@ -205,6 +201,78 @@ private static void updateLibertyServer() throws Exception {
     public void testReturnToPrimaryServer() throws Exception {
         String primaryServerJNDICall = "JNDI_CALL search\\(Name,String,SearchControls\\) \\[ldap://localhost:" + String.valueOf(ds1.getLdapPort()) + "\\]";
 
```

Comment:
Configure the server here instead of in setupClass.

```suggestion
updateLibertyServer(true, 1);
```